collections/brandedColleges.js

```diff
@@ -36,7 +36,8 @@ BrandedColleges.attachSchema(new SimpleSchema({
   enabledFeatures: {type: [String], defaultValue: [], optional: true},
   facebook: {type: new SimpleSchema({
     pageId: {type: String},
-    pageAccessToken: {type: String}
+    pageAccessToken: {type: String},
+    subscriptionMessagingStatus: {type: String, defaultValue: 'not_yet_requested'}
```

Comment:
Should this be optional and/or backfilled?  (I don't remember exactly how optional and default values interact.)

Reply:
I think having a defaultValue makes it irrelevant whether it's optional or not, if I understand correctly. I think it will always be treated as having the default value when it's read in and doesn't exist, and I think it will always be created with the default value.

Reply:
That makes sense.  I just vaguely remember some discussion (possibly in Python-land, not Node-land) about defaults only working on writes, not reads.

Reply:
I've checked the simple-schema documentation, and it confirms.